recourse_methods/mrmc_method.py

```diff
@@ -121,48 +210,61 @@ class Clusters:
     """Class for kmeans cluster-related data.
 
     Attributes:
-        cluster_assignments: A DataFrame containing columns 'datapoint_index' and 'datapoint_cluster'.
+        cluster_assignments: A DataFrame containing columns 'datapoint_index'
+            and 'datapoint_cluster'.
         cluster_centers: A numpy array containing the cluster centers.
     """
+
     cluster_assignments: pd.DataFrame
     cluster_centers: np.ndarray
 
 
 class MRMC(RecourseMethod):
     """A class for MRM with clustering."""
-    def __init__(self,
-                 k_directions: int,
-                 adapter: recourse_adapter.RecourseAdapter,
-                 dataset: pd.DataFrame,
-                 label_column: str = 'Y',
-                 positive_label: Any = 1,
-                 alpha: AlphaFunction = get_volcano_alpha(),
-                 rescale_direction: RecourseRescaler = normalize_rescaler,
-                 clusters: Clusters = None):
+
+    def __init__(
+        self,
+        k_directions: int,
+        adapter: recourse_adapter.RecourseAdapter,
+        dataset: pd.DataFrame,
+        label_column: str = "Y",
+        positive_label: Any = 1,
+        alpha: AlphaFunction = get_volcano_alpha(),
+        rescale_direction: RecourseRescaler = normalize_rescaler,
+        clusters: Clusters = None,
+    ):
         """Creates a new MRMC instance.
 
-        MRMC clusters the data and initializes a separate MRM instance for each cluster.
-        
+        MRMC clusters the data and initializes a separate MRM instance for
+        each cluster.
+
         Args:
-            k_directions: The number of clusters (and recourse directions) to generate.
+            k_directions: The number of clusters (and recourse directions) to
+                generate.
             adapter: The dataset adapter.
             dataset: The dataset to perform recourse over.
             label_column: The column containing binary classification outputs.
             positive_label: The value of a positive classification.
             alpha: The alpha function each MRM should use.
             rescale_direction: The rescaling function each MRM should use.
-            clusters: The cluster data to use. If None, performs k-means clustering.
+            clusters: The cluster data to use. If None, performs k-means
+                clustering.
         """
         X = MRM.process_data(dataset, adapter, label_column, positive_label)
```

Comment:
Better to use `data` instead of `X` (or another more descriptive name) - here and throughout the codebase.